### DISCUSSION TITLE
aerospace does not show up in search on brew.sh

### DISCUSSION THREAD

**Discussion by nyc-de** [nikitabobko/AeroSpace#757] [general]
Not sure if intended or where to put this.  Didn't seem issue worthy.

![Screenshot 2024-11-25 at 9 50 25 AM](https://github.com/user-attachments/assets/ea94eaad-26ec-4054-91e4-bb8101924638)

**Comment by TJTorola** [comment]
Yeah, it's under it's own tap, see in the instructions `brew install --cask nikitabobko/tap/aerospace` which means the source for the package is `https://github.com/nikitabobko/homebrew-tap` rather than the standard repository `https://github.com/Homebrew/homebrew-cask`.
